dev/devicelab/bin/tasks/gradle_plugin_test.dart

```diff
@@ -208,11 +225,49 @@ Future<Null> _runGradleTask({String workingDirectory, String task}) async {
     throw 'Gradle exited with error';
 }
 
-Future<ProcessResult> _resultOfGradleTask({String workingDirectory, String task}) {
+Future<ProcessResult> _resultOfGradleTask({String workingDirectory, String task,
+    List<String> options}) {
+  final List<String> args = <String>['app:$task'];
+  if (options != null) {
+    args.addAll(options);
+  }
   return Process.run(
     './gradlew',
-    <String>['app:$task'],
+    args,
     workingDirectory: workingDirectory,
     environment: <String, String>{ 'JAVA_HOME': javaHome }
   );
 }
+
+class _Dependencies {
+  String target;
+  Set<String> dependencies;
+  _Dependencies(String depfilePath) {
+    final RegExp _separatorExpr = new RegExp(r'([^\\]) ');
+    final RegExp _escapeExpr = new RegExp(r'\\(.)');
```

Comment:
I don't immediately see what the utility of these `RegExp`s are and why we need the semi-complex expression in lines 254-262.

It might help to add a comment or to make the depfile example in line 250 include `some\ file.dart`